components/brave_wallet/browser/brave_wallet_constants.cc

```diff
@@ -163,78 +163,89 @@ const std::vector<mojom::BlockchainToken>& GetRampBuyTokens() {
+       {"0xdeFA4e8a7bcBA345F687a2f1456F5Edd9CE97202", "Kyber Network",
+        "kyber.png", true, false, "KNC", 18, true, "", "",
+        mojom::kMainnetChainId, mojom::CoinType::ETH},
+       {"0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2", "Maker", "mkr.png", true,
+        false, "MKR", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xd26114cd6ee289accf82350c8d8487fedb8a0c07", "OMG Network", "omg.png",
+        true, false, "OMG", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"", "Polygon", "", false, false, "MATIC", 18, true, "", "",
+        mojom::kPolygonMainnetChainId, mojom::CoinType::ETH},
+       {"0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0", "Polygon", "matic.png",
+        false, false, "MATIC", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0x45804880de22913dafe09f4980848ece6ecbaf78", "Pax Gold", "paxg.png",
+        true, false, "PAXG", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0x8e870d67f660d95d5be530380d0ec0bd388289e1", "Pax Dollar", "usdp.png",
+        true, false, "PAX", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0x95ad61b0a150d79219dcf64e1e6cc01f0b64c4ce", "Shiba Inu", "shib.png",
+        true, false, "SHIB", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"", "Solana", "", false, false, "SOL", 9, true, "", "",
+        mojom::kSolanaMainnet, mojom::CoinType::SOL},
+       {"0xdAC17F958D2ee523a2206206994597C13D831ec7", "Tether", "usdt.png",
+        true, false, "USDT", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xc944e90c64b2c07662a292be6244bdf05cda44a7", "The Graph",
+        "graphToken.png", true, false, "GRT", 18, true, "", "",
+        mojom::kMainnetChainId, mojom::CoinType::ETH},
+       {"0x3845badAde8e6dFF049820680d1F14bD3903a5d0", "TrueUSD", "tusd.png",
+        true, false, "TUSD", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0x0000000000085d4780B73119b644AE5ecd22b376", "The Sandbox", "sand.png",
+        true, false, "SAND", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984", "Uniswap", "uni.png",
+        true, false, "UNI", 18, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", "USD Coin", "usdc.png",
+        true, false, "USDC", 6, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xe41d2489571d322189246dafa5ebde1f4699f498", "Ox", "zrx.png", true,
+        false, "ZRX", 6, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH},
+       {"0xbb0e17ef65f82ab018d8edd776e8dd940327b28b", "Axie Infinity",
+        "axs.png", true, false, "AXS", 6, true, "", "", mojom::kMainnetChainId,
+        mojom::CoinType::ETH}});
```

Comment:
Please review all the decimals correctly. ZRX and AXS should have `6`.

Reply:
@onyb  Fixed here: 4027fca7790bb2df255964f4fd6f23c98632a743